### ISSUE TITLE
demos/imagenet/ does not work on one system.

### ISSUE THREAD

**Issue by MariasStory** [tensorflow/tfjs-core#496]
https://deeplearnjs.org/demos/imagenet/
Ubuntu 17.10,
oibaf video drivers,
Intel Corporation 3rd Gen Core processor Graphics Controller (rev 09) (prog-if 00 [VGA controller]),
Unity interface,
Chrome and Firefox

constantly showing:
```
tennis ball     1
cliff dwelling  0
...             0
```
No errors in console.

**Comment by MariasStory** [comment]
The same in Wayland interface on this system.

I have similar setup with Radeon R7 and it works there without problems.

**Comment by nsthorat** [comment]
@dsmilkov does this possibly have to do with the thing you were seeing on Video elements?

**Comment by dsmilkov** [comment]
Unclear. @MariasStory did you use the webcam, or one of the predefined images on the demo?

**Comment by MariasStory** [comment]
Hi @dsmilkov, I did use webcam. So far, I have tried Chrome 64 and 65, also Firefox 57.

**Comment by dsmilkov** [comment]
Hi @MariasStory, we had a guess of what might be wrong and we updated https://deeplearnjs.org/demos/imagenet/.

Curious if you still see the issue? (Make sure you hard refresh/clean browser cache to get the new js code)

Thanks

**Comment by MariasStory** [comment]
Hi @dsmilkov, unfortunately, the demo does not work :-(.
I did "empty cache and hard reload".
Is there something else that I can collect on my system to help in solving the problem?

**Comment by MariasStory** [comment]
Hi, sorry to come back to this issue, but I have additional results from the system:
Using https://deeplearnjs.org/demos/benchmarks/
batchnorm3d_gpu is mostly 0.100ms and in one case 0.000ms
mulmat_gpu s mostly 0.100ms, 4 times 0.000ms and in one case 0.200ms
CPU values seem to be normal. I guess it is because of the optimized "olaf" drivers or the forced GPU rendering in Chrome. Any way, this example shows that there should be a test for the realistic results from GPU rendering.
